SpywareFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||tebilisim.com/analytics.js
```

Comment:
let's use `||cdn.tebilisim.com/analytics.js` because this rule will not match this `https://te-tebilisim-com.cdn.tebilisim.com/analytics.js?v=1.56.2` URL in Safari

Reply:
So what about these issues?
#104098
#104100
#104102
`tebilisim` is there too

Reply:
@gork7777 
Let's add `/_ajax/tebilisim.php?`  for them in general_url

Reply:
@gork7777 to general_url in tracking protection (sorry, I forgot to say)